Expensify/App
# Added Keyboard shortcuts modal

### Details

- Added help dialog for keyboard shortcuts list

### Fixed Issues

$ https://github.com/Expensify/App/issues/5588

### Tests
1. Checked keyboard shortcut with Desktop and Web
2. Checked from Windows Browser for Ctrl + ? shortcut.
3. Compared all the current keyboard shortcuts.



### QA Steps
1. On the sidebar click `Cmd + ?`
2. It should open a dialog with the list of keyboard shortcuts.
3. It should cover all the current keyboard shortcuts.



### Tested On


### Screenshots


#### Web

<img width="1440" alt="Screenshot 2021-11-08 at 9 28 02 PM" src="https://user-images.githubusercontent.com/57435789/140775191-754b7d85-7fb0-4bd9-9ad0-426a2375bc08.png">
<img width="359" alt="mobile-view-shortcuts-modal" src="https://user-images.githubusercontent.com/57435789/140796568-36307d81-2aea-457e-b390-8d1e2ab9b796.png">



#### Mobile Web


#### Desktop

<img width="1197" alt="Screenshot 2021-11-08 at 9 30 37 PM" src="https://user-images.githubusercontent.com/57435789/140775578-54d7f45b-1414-49be-a72a-d1c32cd974e8.png">

#### iOS


#### Android
